web-console/5-components/quota-warnings.md

```diff
@@ -0,0 +1,58 @@
+# Quota Warnings
+
+## General
+- Distinguish between **compute-resource** limits and **object-count** quotas.
+- Both types of quotas/limits should be further broken down to specify the resource in question, e.g. Pods, Services, PVCs, Routes, Secrets, CPU, Memory.
+- Messages should be customizable.
+
+![quota kebab](img/quota-kebab.png)
+
+- Toasts and notification drawer messages should include a kebab menu with options to:
+  - View Quotas
+  - Don't Show Me Again
+  - Close notification
```

Comment:
@ncameronbritt Should this last option actually be "Clear?" That's the action on the right side of the other rows and also the syntax used for the bottom action link, "Clear All."

Reply:
Makes sense to me. Would that be the same for toasts too?